fh_immuta_utils/data_source.py

```diff
@@ -53,11 +53,12 @@ def make_immuta_datasource_name(
     Returns a table name that's guaranteed to be unique and within the Immuta data source name max char limit (255)
     """
     table_name = ""
-    if user_prefix:
-        table_name = f"{user_prefix}_"
-    table_name += f"{PREFIX_MAP[handler_type]}_{schema}_{table}"
-    if table_name is None:
-        return None
+    table_name += f"{user_prefix}_" if user_prefix else ""
+    table_name += f"{PREFIX_MAP[handler_type]}_" if handler_type else ""
+    table_name += f"{schema}_" if schema else ""
+    table_name += f"{table}"
+    if not table_name:
```

Comment:
Nit, is this check necessary? Rest of the code seems to already be doing this.

Reply:
removed!